pkg/services/users/users_test.go

```diff
@@ -0,0 +1,150 @@
+package users
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hakierspejs/long-season/pkg/models"
+	"github.com/matryer/is"
+)
+
+func Test_Equals(t *testing.T) {
```

Comment:
Hmmm, strange things are going on here. I thought that you had changed functions names to proper camelCase.

Everything else is looking very good :). Please change these function names to proper CamelCase and it will be ready to merge.

Reply:
btw thank you for using `is` package :)